hackathon_site/dashboard/frontend/src/pages/Teams/Teams.tsx

```diff
@@ -20,9 +35,14 @@ const Teams = () => {
             key={index}
             className={styles.teamsListGridItem}
             grid-template-column="true"
-            onClick={() => alert(`Goes to team-detail for team ${team.TeamName}`)}
+            onClick={() => alert(`Goes to team-detail for team ${team.team_code}`)}
         >
-            <TeamCardAdmin teamCode={team.TeamName} members={team.Members} />
+            <TeamCardAdmin
+                teamCode={team.team_code}
+                members={team.profiles.map(
```

Comment:
nit pick:
here you .map() the profiles but in TeamCardAdmin you do .map() again on the member prop so to write more concise code you could make members a Profile array type and perform .map() just once.

Not a big deal tho, don't have to change.

Reply:
sorry eric i changed my mind, could you fix this please 🙏